bats/tests/k8s/enable-disable-k8s.bats

```diff
@@ -0,0 +1,41 @@
+# Test case 8 & 13
+
+setup() {
+    load '../helpers/load'
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+verify_k8s_is_running() {
+    wait_for_container_engine
+    run kubectl cluster-info
+    run rdsudo "$RC_SERVICE" --nocolor k3s status
+    assert_success
+    assert_line --partial "status: started"
+}
+
+@test 'start rancher desktop with kubernetes enabled' {
+    start_kubernetes
+    wait_for_apiserver
+    verify_k8s_is_running
+}
+
+@test 'disable kubernetes' {
+    rdctl set --kubernetes-enabled=false
+    wait_for_container_engine
+    echo "Sleeping for 60 seconds before testing to see if k3s is running..." 1>&3
+    sleep 60
```

Comment:
Not a big fan of using `sleep` in code, but maybe appropriate here. We can review later if there is a better way to do this (can't think of anything right now).